package.json

```diff
@@ -141,14 +148,20 @@
     "@lcdp/offline-plugin": "^5.1.0",
     "@lhci/cli": "0.8.x",
     "@playwright/test": "^1.27.1",
-    "@storybook/addon-actions": "^5.2.1",
-    "@storybook/addon-knobs": "^5.2.1",
-    "@storybook/addon-links": "^5.2.1",
-    "@storybook/addons": "^5.2.1",
-    "@storybook/react": "^5.2.1",
+    "@storybook/addon-a11y": "^6.5.16",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
```

Comment:
Why are there additional addons ?
@storybook/addon-actions, @storybook/addon-knobs, @storybook/addon-links iwere the ones that were there earlier. Lets just keep them unless we are doing something with the new plugins that are added

Reply:
@storybook/knobs is deprecated  rest are added by storybook itself.